src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,46 @@
-export const Tabs = () => {};
+import { Tab } from '../../types/Tab';
+
+type Props = {
+  tabs: Tab[];
+  selectedTabId: string;
+  onTabSelected: (tab: Tab) => void;
+};
+
+export const Tabs: React.FC<Props> = (props) => {
+  const { tabs, selectedTabId, onTabSelected } = props;
+  const selectedTab = tabs.find(tab => tab.id === selectedTabId) || tabs[0];
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(tab => (
+            <li
+              className={
+                selectedTab?.id === tab.id
+                  ? 'is-active'
+                  : ''
```

Comment:
Better install [classnames](https://www.npmjs.com/package/classnames) and use it instead of the ternary operator.
The ternary operator is very limited in its functionality to be used in classNames. 
[classnames](https://www.npmjs.com/package/classnames) usually used in real projects and would be great to work with it now and would be needed to do in test tasks.